spec/unit/ConfigChanges.spec.js

```diff
@@ -187,12 +187,14 @@ describe('generate_plugin_config_munge for windows project', function() {
         // 1 comes from versions="=8.1.0" + 1 from versions="=8.1.0" device-target="windows"
         expect(windows81AppxManifest.parents['/Parent/Capabilities'][0].xml).toBe('<Capability Note="should-exist-for-all-appxmanifest-target-files" />');
         expect(windows81AppxManifest.parents['/Parent/Capabilities'][1].xml).toBe('<Capability Note="should-exist-for-win81-win-and-phone" />');
+        expect(windows81AppxManifest.parents['/Parent/Capabilities'][1].count).toBe(2);
```

Comment:
@matrosov-nikita is this a right element to check count?
Should not it be `expect(windows81AppxManifest.parents['/Parent/Capabilities'].count).toBe(4);`?
I know it was this way in the original code, but it just looks strange.